### PR TITLE
solution

### PR DESCRIPTION
**Pull request by ivan-shpynda**
[DEMO LINK](https://ivan-shpynda.github.io/react_goods-selector/)

### CODE REVIEW

**Review by Grygoriy-Shytikov — APPROVED** [review]
In my opinion, it looks good! 👍

File: src/App.js [hunk 1]
```diff
@@ -14,11 +14,59 @@ const goodsFromServer = [
...
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    goods: ['Jam'],
```
Comment (varseniuk) [inline src/App.js:19]:
```suggestion
    selectedGoods: ['Jam'],
```

File: src/App.js [hunk 2]
```diff
@@ -14,11 +14,59 @@ const goodsFromServer = [
...
+        <div className="top">
+          <h1>
+            {goods.length === 0
+              ? 'No goods selected'
+              : (`${goods.map(good => (` ${good}`))} selected`)}
+          </h1>
+          {goods.length === 0
+            ? ''
+            : (
+              <button
```
Comment (varseniuk) [inline src/App.js:36]:
```suggestion
          {!!goods.length && (
            <button
```
Let's use conditional rendering like this

File: src/App.js [hunk 3]
```diff
@@ -14,11 +14,59 @@ const goodsFromServer = [
...
+        {goodsFromServer.map(good => (
+          <div
+            key={good}
+            className={goods.includes(good) ? 'active good' : 'good'}
+          >
+            {good}
+            {goods.includes(good)
+              ? ''
+              : (
+                <button
```
Comment (varseniuk) [inline src/App.js:56]:
```suggestion
            {!goods.includes(good) && (
              <button
```

File: src/App.js [hunk 4]
```diff
@@ -14,11 +14,59 @@ const goodsFromServer = [
...
+                  type="button"
+                  onClick={() => {
+                    this.setState({ goods: [...goods, good] });
+                  }}
+                >
+                  Select
+                </button>
+              ) }
```
Comment (varseniuk) [inline src/App.js:64]:
```suggestion
              )}
```

**Review by varseniuk — CHANGES_REQUESTED** [review]
Good work!
Let's fx some issues with conditional rendering syntax and list rendering

**Review by varseniuk — APPROVED** [review]

**Pull request closed without merging**